hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/test/TestSources.java

```diff
@@ -161,4 +187,59 @@ void fillBuffer(TimestampedSourceBuffer<T> buf) throws Exception {
             }
         }
     }
+
+    /**
+     * Returns a {@link com.hazelcast.jet.pipeline.StreamSource} for {@code
+     * long} values. The value creation is distributed across the cluster and
+     * is done using {@link StreamSourceLong}. All {@link StreamSourceLong}
+     * instances are created identically, one per cluster member. The returned
+     * source is designed to be used for high-throughput performance testing.
+     *
+     * @param itemsPerSecond how many items should be emitted each second
+     * @param initialDelay initial delay before emitting values
```

Comment:
what unit is this in? is it in real-time or event time?

Reply:
Unit is added in d476f97f6c1cb3bdd945b2b0da124869086022da.
 It is real-time, however, it only affects the earliest timestamp that is going to be emitted.